README.md

```diff
@@ -1,24 +1,97 @@
+    + [HMDA Filing](#hmda-filing)
+    + [HMDA Data Browser](#hmda-data-browser)
+    + [HMDA Platform Tools](#hmda-platform-tools)
+    + [HMDA Documentation](#hmda-documentation)
+    + [HMDA Data Publication](#hmda-data-publication)
+  * [Development](#development)
+    + [Requirements](#requirements)
+    + [Installation](#installation)
+    + [Running Locally](#running-locally)
+    + [Running via Docker](#running-via-docker)
+  * [Testing](#testing)
+    + [Unit Tests](#unit-tests)
+    + [End-to-End Testing](#end-to-end-testing)
+
+## Technical Overview
+Each React application lives in it's own sub-directory of the `/src` folder, with shared assets and components housed in `/common`. All sub-applications are rendered as asynchronous components within an application shell `/App.jsx` that provides a common header.  This approach eliminates unneccesary reloading of the site-wide navigation, giving the separate apps a more connected feel.  [React Router](https://reacttraining.com/react-router/) is used for client side routing with [React Redux](https://redux.js.org/) integrated for state management of the more complex apps, such as Filing.  [Unit tests](#unit-tests) are developed using Enzyme and [end-to-end](#end-to-end-testing) testing performed with Cypress. Dependencies are managed with [yarn](https://classic.yarnpkg.com/en/).
+
+## Applications
+
+### HMDA Homepage
+[HMDA Homepage](https://ffiec.cfpb.gov/) is a single-page app providing easy access to the HMDA applications, tools, data products, and documentation.
+
+<a href='./readme-files/hmda-homepage.png' alt='HMDA Homepage'>
+  <p align='center'>
+    <img src='./readme-files/hmda-homepage.png' width='80%'  overflow='scroll'/>
+  </p>
+</a>
+
+### HMDA Filing
+The [HMDA Filing Platform UI](https://ffiec.cfpb.gov/filing/) allows lending institutions to submit HMDA records, resolve errors, verify edits, review submission status and history, and sign submissions.
+
+<a href='./readme-files/hmda-filing.png' alt='HMDA Filing Overview'>
+  <p align='center'>
+    <img src='./readme-files/hmda-filing.png' width='80%'  overflow='scroll'/>
+  </p>
+</a>
+
+
+### HMDA Data Browser
+The [HMDA Data Browser](https://ffiec.cfpb.gov/data-browser/) enables users to easily filter and download aggregated HMDA datasets.  
+
+<a href='./readme-files/hmda-data-browser.gif' alt='HMDA Data Browser'>
+  <p align='center'>
+    <img src='./readme-files/hmda-data-browser.gif' width='80%'  overflow='scroll'/>
+  </p>
+</a>
+
+### HMDA Platform Tools
+The [HMDA Platform Tools](https://ffiec.cfpb.gov/tools/) assist filers in the preparation of submission data, including calculation of Rate Spread, generation and validation of Check Digits, as well as submission file generation and format verification.
+
+<a href='./readme-files/hmda-tools.png' alt='HMDA Platform Tools'>
+  <p align='center'>
+    <img src='./readme-files/hmda-tools.png' width='80%'  overflow='scroll'/>
+  </p>
+</a>
+
+### HMDA Documentation
```

Comment:
mentioning that hmda-documentation is rendered by markdown files in this repo would be nice to add here.